### ISSUE TITLE
Cannot open local file from pull request review.

### ISSUE THREAD

**Issue by michalsustr** [TeamCodeStream/codestream#334]
Hi, this is my first time checking out codestream, I just found it so I wanted to give it a try. Thanks for open-sourcing the project!

I am using Intellij's Clion 2020.1.2. 
I cannot open a local file from a pull request review  (by clicking on the small icon in the corner), even though I am on the same git branch. 

![image](https://user-images.githubusercontent.com/1756494/99854235-aff6fc80-2b41-11eb-87d4-dc20cdcc469d.png)

I am not sure what relevant information I can give you to help fix this bug -- maybe you can give me some pointers?

Also the fonts are quite broken as you can see in the screenshot, should I post another issue for this?

**Comment by planteater** [comment]
@michalsustr - Thanks for reaching out on this. I'll follow up later on the font issue, but for the Open Local File issue a log file would be great. Reproduce the issue and then in your IDE go to Help > Collect Logs and Diagnostic Data. This will open the finder where you should see a newly created zip file. If you'd prefer not to share that file here, you can send it to me at support@codestream.com. Thanks!

**Comment by planteater** [comment]
@michalsustr - Just wanted to follow up to see if we could get that log file. Thanks!

**Comment by michalsustr** [comment]
Hmm, now the opening of the files does work so I can't replicate the issue. The fonts still look the same as on the screenshot.

**Comment by planteater** [comment]
@michalsustr - I tried reaching you via email regarding the display issue, but I must not have the correct one.

From your original report you indicated that you were using CLion Clion 2020.1.2. I was wondering if there was any possibility of updating to 2020.2 (or later). With 2020.2 JetBrains IDE have native functionality for rendering webviews (JCEF). In earlier versions, like yours, we had to use a 3rd party extension called JxBrowser and that's probably behind your display issues.

If you are update to update, after doing so go to Preferences > Tools > CodeStream and make sure the box to use JCEF Webview is checked, and then restart your IDE if it wasn't.

Thanks!

(Email me at support@codestream.com if you want to move this issue to email.)
